src/index.tsx

```diff
@@ -33,7 +35,9 @@ export default class ReactToPrint extends React.Component<IReactToPrintProps> {
         setTimeout(() => {
             target.contentWindow.focus();
             target.contentWindow.print();
-            
+            if (this.props.removeAfterPrint) {
```

Comment:
I think this should come after the `if` below, so that `onAfterPrint` can interact with the `iframe` if the user needs it to?

Reply:
You are right - I moved it to be after onAfterPrint.